frontend/src/PagesAdmin/InterviewNotesAdminPage/InterviewNotesAdminPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { Button } from '~/Components';
+import { AdminPageLayout } from '../AdminPageLayout/AdminPageLayout';
+import styles from './InterviewNotesAdminPage.module.scss';
+
+export function InterviewNotesPage() {
+  //TODO: interview notes from backen
+  const [editingMode, setEditingMode] = useState(false);
+  const [text, setText] = useState('Notater fra intervjuet her...'); //TODO: place the text from the backend here.
+  const posid = 1; //TODO: get the posid from the backend.
+
+  function handleOnClick() {
+    if (editingMode) {
+      setEditingMode(false);
+      //TODO: Save the text in the textbox and send it to the backend.
+    } else {
+      setEditingMode(true);
+      //TODO: Get the text from the backend, and place it in the textbox.
+    }
```

Comment:
Kan gjøres mer kompakt.
```suggestion
    if (editingMode) {
      setEditingMode(!editingMode);
      //TODO: Save the text in the textbox and send it to the backend.
      //TODO: Get the text from the backend, and place it in the textbox.
    }
```

Reply:
Gjorde det mer kompakt på en litt annen måte enn du foreslo